### PR TITLE
tests: unicode all the way

### PR DESCRIPTION
**Pull request by greut**
Using utf-8 strings in the tests. It's easier to check that the output correspond when you don't have to match a character with it's html entity.

Remove some non used stuff and flake warnings.

And break the tests ;-)

### CODE REVIEW

**Comment by greut** [comment]
@jirikuncar I've removed the culprit. Let me know what needs to be changed before you can consider merging it. Thanks!

**Comment by jirikuncar** [comment]
@greut can you check situation in which a field contains any html entity like `&gt;`?

## 

`s/test:/tests:/` :wink:

**Comment by greut** [comment]
Added in the test suite: `from __future__ import unicode_literals`

**Comment by jirikuncar** [comment]
@greut `from __future__ import unicode_literals` is not good idea. Can you use `u` explicitly instead?

File: tests/test_core.py [hunk 1]
```diff
@@ -9,14 +9,13 @@
 
 """Test suite for DoJSON."""
 
+from __future__ import unicode_literals
```
Comment (jirikuncar) [inline tests/test_core.py]: please use `u` instead

**Comment by greut** [comment]
@jirikuncar m'kay.

**Pull request merged**